src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -128,10 +129,14 @@ export default Backbone.Model.extend({
       },
     };
 
-    if (this.id === 'shared-by' || !clinicianId) {
-      filters[this.id].role = roleId;
+    if (noOwner) {
```

Comment:
I don't think this is correct.  We still want to filter by the role, we just want to add the clinician filter.  so this `if (noOwner)` should be inside the `if(this.id === 'shared-by' ||...` such that role and clinician are both set.